frontend/src/pages/org/browser-profiles-list.ts

```diff
@@ -101,7 +102,18 @@ export class BrowserProfilesList extends LiteElement {
       >
         <div class="grid grid-cols-8 gap-3 md:gap-5" role="row">
           <div class="col-span-8 md:col-span-3 p-2" role="cell">
-            <div class="font-medium mb-1">${data.name}</div>
+            <div class="font-medium flex items-end">
```

Comment:
```suggestion
            <div class="font-medium text-sm">
```
The `text-sm` here aligns the text vertically in the container correctly:

With:
![localhost_9870_orgs_default-org_browser-profiles](https://github.com/webrecorder/browsertrix-cloud/assets/5727389/066f1486-3d08-4599-8344-bd8e222f8608)

Without:
![localhost_9870_orgs_default-org_browser-profiles (1)](https://github.com/webrecorder/browsertrix-cloud/assets/5727389/43bdb60b-9c04-4ca9-9aed-3579cb5ac6b3)